imap_processing/tests/ultra/unit/test_lookup_utils.py

```diff
@@ -0,0 +1,68 @@
+import numpy as np
+import pandas as pd
+
+from imap_processing import imap_module_directory
+from imap_processing.ultra.l1b.lookup_utils import (
+    get_back_position,
+    get_energy_norm,
+    get_image_params,
+    get_norm,
+    get_y_adjust,
+)
+
+base_path = imap_module_directory / "ultra" / "lookup_tables"
```

Comment:
Global constants should be all uppercase.
```suggestion
BASE_PATH = imap_module_directory / "ultra" / "lookup_tables"
```